src/test/tools/convert_xml_to_yaml/yaml2xml.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" ?>
+<shadow stoptime="3600">
```

Comment:
Again would be nice to preserve our ~canonical ordering. Maybe less important here though since the idea is that generally humans won't be reading the xml file.

Reply:
I didn't remove the  doctype `<?xml version="1.0" ?>`. I didn't find an elegant way to do it... What do you think ? Should I hard code a solution to remove it ?

Reply:
As long as shadow still parses it, I think it's fine.